### PR TITLE
Fixes on the "estados de la póliza" page

### PR DESCRIPTION
**Pull request by amatmv**
### Description:

Various fixes to the "Estados de la póliza" page in Spanish.

### Tasks:

- [x] Fixed wrong link to file name that caused the workflow diagram could not be seen in Spanish. 
- [x] Wrong link to "modificaciones contractuales" page. 

### Links:

* [es/base/polizas/#estado-de-las-polizas](http://builds.gisce.net/powerp-docs/powerp_fix_wrong_image_name/es/base/polizas/#estado-de-las-polizas)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_fix_wrong_image_name/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_fix_wrong_image_name/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: locales/es_ES/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -3556,7 +3556,7 @@ msgid ""
 "explicació més detallada d'aquestes, consultar\n"
 "\n"
 "<a href=\"#modificacions-contractuals_1\">aquesta</a> pàgina."
-msgstr "Desde Activa también se pueden hacer <strong>Modificaciones Contractuales</strong>. Para una explicación más detallada de éstas, consultar <a href=\"../#modificaciones-contractuales_1\">esta</a> página."
+msgstr "Desde Activa también se pueden hacer <strong>Modificaciones Contractuales</strong>. Para una explicación más detallada de éstas, consultar <a href=\"#modificaciones-contractuales_1\">esta</a> página."
```
Comment (MiquelIR) [inline locales/es_ES/LC_MESSAGES/messages.po]: This link is not working for me

**Comment by jaumef** [comment]
@amatmv are you actually WIP or we can merge? :stuck_out_tongue:

**Comment by amatmv** [comment]
@jaumef Can be merged!

**Pull request merged**